front/src/App.js

```diff
@@ -162,12 +162,8 @@ class App extends Component {
         <Container>
           {activePage === "refbox" && (
             <div>
-              <RefboxField onClick={(tag, teamColor) => {}} />
-              <RefboxSettings
-                onClickDropBall={() => {}}
-                onClickInOut={prop => {}}
-                onChange={(type, value) => {}}
-              />
+              <RefboxField isPenalty={false} />
```

Comment:
Does this just disable penalty move completely?

Reply:
Not quite sure. We should have a branch for completely using the penalty box or something. It still exists, at the very least.